packages/gherkindoc/test/ProgramDevCLI.feature

```diff
@@ -0,0 +1,153 @@
+Feature: SaaSquatch Program Development CLI
+
+    The SaaSquatch program dev CLI aims to improve the workflow of a program developer by automating
+    the process of uploading code/schemas to Webtask and Contentful.
+
+    @approved
+    @implemented
+    Scenario: Sensitive information will be loaded through environment variables
+        Given I have installed the CLI
+        When the CLI loads and begins executing
+        Then if a .env file is found in the current working directory it will be loaded
+        And the environment variables will store the sensitive information
+
+    @review
+    @implemented
+    Scenario: Users can log in via the CLI
+        Given I have just installed the CLI and want to log in
+        And I have accounts on Contentful and Webtask with the proper access rights
```

Comment:
Does this file still need to be here, since the CLI was removed?

Reply:
The CLI was removed but that file is there just so there is something to test the generator stuff with

Reply:
Sounds good, can it be cleaned up? It's still referencing some stuff that isn't relevant anymore like webtask.

Reply:
I mean I suppose it could, but does it really matter if it's just being used for testing the picklesdoc clone? No one is using this as an actual spec